### ISSUE TITLE
GNUInstallDirs generates warnings outside Linux

### ISSUE THREAD

**Issue by lemire** [simdjson/simdjson#769]
On freebsd and macOS, one gets the following warning:

```
CMake Warning (dev) at /usr/local/Cellar/cmake/3.17.1/share/cmake/Modules/GNUInstallDirs.cmake:225 (message):
  Unable to determine default CMAKE_INSTALL_LIBDIR directory because no
  target architecture is known.  Please enable at least one language before
  including GNUInstallDirs.
Call Stack (most recent call first):
  CMakeLists.txt:2 (include)
This warning is for project developers.  Use -Wno-dev to suppress it.
```

**Comment by lemire** [comment]
(I don't know whether it is a problem but users are likely to complain about it.)

**Comment by jkeiser** [comment]
This was fixed with #788.
